client/client.go

```diff
@@ -14,7 +14,7 @@ import (
 	"github.com/pkg/errors"
 )
 
-const maxRetries = 3
```

Comment:
could it be an idea to be able to customize this via command line arguments? 💡

Reply:
Ah yeah good shout! I just whacked it up for now as it didn't seem harmful to just try a bit more 😄